### DISCUSSION TITLE
Ubuntu 24.04 repositories?

### DISCUSSION THREAD

**Discussion by muja** [nodesource/distributions#1807] [Ideas]
Any plans on adding Ubuntu 24.04 (noble) repositories? I couldn't find any issues/discussions about this which makes me wonder if this is out of scope for this repo? If so, where would I need to look?

Thanks

**Comment by JesusPaz** [comment]
Hello, all versions of Ubuntu are supported by default. You only need to setup the repository using our [script](https://github.com/nodesource/distributions?tab=readme-ov-file#installation-instructions-deb).
